app/components/timeline_component/timeline_component.scss

```diff
@@ -42,7 +42,7 @@
         width: 5px;
       }
 
-      &__key {
+      .timeline-component__key {
```

Comment:
why remove the &__key notation?

Reply:
& inside sass modules/components or whatever makes the code more flexible if the wrapper naming were to change

Reply:
because it wouldn't make sense anymore. The class isn't `timeline-component__item__key`

Reply:
the class and/or structure of this sass is incorrect then, but willing to let it go as that happened somewhere along the way before this PR